editor/project_manager.cpp

```diff
@@ -1678,6 +1681,48 @@ void ProjectList::erase_missing_projects() {
 	save_config();
 }
 
+void ProjectList::_scan_folder_recursive(const String &p_path, List<String> *r_projects) {
+	Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	Error error = da->change_dir(p_path);
+	ERR_FAIL_COND_MSG(error != OK, vformat("Failed to open the path \"%s\" for scanning with code %d.", p_path, error));
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(error != OK, vformat("Failed to open the path \"%s\" for scanning. Error code %d.", p_path, error));
```
"scanning with code" sounds ambiguous.

Reply:
It's "Failed ... with code", we use this structure elsewhere (though, admittedly, it was I who added it elsewhere, probably)

Reply:
Rephrased it a bit.